torchgeo/datasets/soda.py

```diff
@@ -0,0 +1,353 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""SODA-A dataset."""
```

Comment:
```suggestion
"""SODA datasets."""
```
Someday we may add SODA-D to this file